ecocodejava.properties

```diff
@@ -15,7 +14,13 @@ defaults.mavenArtifactId=ecocode-java-plugin
 1.2.1.changelogUrl=https://github.com/green-code-initiative/ecoCode/releases/tag/1.2.1
 
 1.3.0.description=Update rules tags�; upgrade librairies to SonarQube 10.0.0�; clean-up plugins and dependencies
-1.3.0.sqVersions=[9.9,LATEST]
+1.3.0.sqVersions=[9.9,10.1]
```

Comment:
This needs to end with the current version (10.3). Otherwise your users on current versions won't be prompted to upgrade.

Reply:
Actually, we have to check that our plugin is ok with new Sonarqube versions i.e. 10.2 and 10.3 before updating this information.
Do you mean that our plugin compatibility must be ok with 10.2 and 10.3 SonarQube versions before submitting it to you ?

Reply:
The old/current value here is `LATEST`. That mean's it's already marked compatible with 10.2 & 10.3. When you archive a plugin, the Marketplace uses the version's compatibility range to know which instances should be prompted to upgrade to the new version. If you end this range with 10.1, your current users already using this version on 10.2 won't be prompted to upgrade. They'll sit on plugin v1.3.0 forever (or until they upgrade SonarQube).

Reply:
I'm going to check compatibility and come back to you soon.

Reply:
@ganncamp it's ok now for me ... I've just add a new commit to my PR
Are you agree with it ?